wait-for-it.sh

```
@@ -0,0 +1,182 @@
+    exit $WAITFORIT_RESULT
+else
+    if [[ $WAITFORIT_TIMEOUT -gt 0 ]]; then
+        wait_for_wrapper
+        WAITFORIT_RESULT=$?
+    else
+        wait_for
+        WAITFORIT_RESULT=$?
+    fi
+fi
+
+if [[ $WAITFORIT_CLI != "" ]]; then
```

Shellcheck (0.10.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)